documentation/optee_design.md

```diff
@@ -523,8 +523,104 @@ implementation, as explained in [crypto.md].
 TBD
 
 # 12. Trusted Applications
-## Format
-Trusted Applications consists of a signed ELF file.
+
+## Static and Dynamic TAs
+
+There are two ways to implement Trusted Applications, static and dynamic.
+For most cases, Dynamically-loaded TAs are preferred.
```

Comment:
Mmmm well, I am merely a native English speaker, but my wife has a degree in it: she says no problem as it is, capitalize the Loaded as well, or as you suggest.